nx-libs.spec

```diff
@@ -510,7 +510,6 @@ rm -r %{buildroot}%{_includedir}/nx-X11/Xtrans
 %dir %{_sysconfdir}/nxagent
 %config(noreplace) %{_sysconfdir}/nxagent/keystrokes.cfg
 %config(noreplace) %{_sysconfdir}/nxagent/nxagent.keyboard
-%config(noreplace) %{_sysconfdir}/nxagent/rgb
```

Comment:
@opoplawski is that enough to drop config files? I searched around, but it seems that RPM has no special concept of config file deprecation. Will it do the right thing automatically, or just leave the file around, even if unmodified?